### ISSUE TITLE
Does not work well with horizontal scroll is iOS Safari

### ISSUE THREAD

**Issue by Jokero** [willmcpo/body-scroll-lock#61]
When body scroll is locked, target element with horizontal scroll is scrolled very jerky, it's frozen at random moments. There is no such issue in Chrome.
I tried with your demo, just changed the width of target element and its child.

**Comment by alexpalchikovskiy** [comment]
@Jokero how you fixed it? I have same problem.

**Comment by Jokero** [comment]
@alexpalchikovskiy No. In order to support horizontal scroll the library has to be patched

**Comment by gblakeman** [comment]
Just ran into this as well… Has anyone successfully patched a version for horizontal scrolling in iOS?

**Comment by wowczarczyk** [comment]
I ended up using the `position: fixed` approach and restoring the scroll position on modal close.

**Comment by xanido** [comment]
Also ran in to this, see willmcpo/body-scroll-lock#133

**Comment by Aarbel** [comment]
Same problem here, @willmcpo  can you fix it ?
